apps/site/src/app/(main)/portal/@applicant/ConfirmAttendance.tsx

```diff
@@ -0,0 +1,36 @@
+import { PortalStatus } from "./ApplicantPortal";
+import Button from "@/lib/components/Button/Button";
+
+interface ConfirmAttendanceProps {
+	status: string;
```

Comment:
Nitpick: It might be better to have the `status` as type `PortalStatus` for consistency with `SignWaiver`.